plugin/othttp/handler.go

```diff
@@ -150,11 +150,9 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if sc.IsValid() { // not a valid span context, so no link / parent relationship to establish
 		var opt trace.SpanOption
 		if h.public {
-			// TODO: If the endpoint is a public endpoint, it should start a new trace
-			// and incoming remote sctx should be added as a link
-			// (WithLinks(links...), this option doesn't exist yet). Replace ChildOf
-			// below with something like: opt = trace.WithLinks(sc)
-			opt = trace.ChildOf(sc)
+			// If the endpoint is a public endpoint, it should start a new trace
+			// and incoming remote sctx should be added as a link.
+			opt = trace.LinkedTo(sc)
 		} else { // not a private endpoint, so assume child relationship
 			opt = trace.ChildOf(sc)
```

Comment:
On this note, aren't we removing `ChildOf`? I'm on the fence about this, because we still haven't fully clarified `SpanKind`. https://github.com/open-telemetry/opentelemetry-specification/pull/337

Reply:
In either case we need a way to set the parent relationship, regardless of whether it's _precisely_ specified according to `SpanKind` or more generic.

I am fine with punting this until we more fully remove ChildOf and just updating in place when we do so.